src/AppBridgeTheme.ts

```diff
@@ -26,6 +32,54 @@ import type {
 export class AppBridgeTheme {
     constructor(private readonly projectId: number) {}
 
+    public async createDocument<D extends Document>(document: D): Promise<Document> {
+        try {
+            return await createDocument<D>(document);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async updateDocument<D extends Document>(document: D): Promise<Document> {
+        try {
+            return await updateDocument<D>(document);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async createDocumentGroup(documentGroup: DocumentGroup) {
+        try {
+            return await createDocumentGroup(documentGroup);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async updateDocumentGroup(documentGroup: DocumentGroup) {
+        try {
+            return await updateDocumentGroup(documentGroup);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async createDocumentPage(page: DocumentPage) {
+        try {
+            return await createDocumentPage(page);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async updateDocumentPage(page: DocumentPage) {
```

Comment:
Same for the others functions
```suggestion
    public async updateDocumentPage(documentPage: DocumentPage) {
```